letta/utils.py

```diff
@@ -488,6 +488,9 @@ def is_utc_datetime(dt: datetime) -> bool:
 
 
 def get_tool_call_id() -> str:
```

Comment:
@sarahwooders should we make this be similar to the other slugs we're generating?

At a certain point in time, we had to do `[:TOOL_CALL_ID_MAX_LEN]` I believe because OpenAI changed (lowered) the max tool ID length on their API which broke a bunch of things, so we probably want to be wary of things like that happening in the future.

We can just take OpenAI's `tool_call_id` as ground truth, though maybe we want to use our own ID format to preemptively be more portable with other providers?

FYI, OpenAI's tool IDs look like `call_xxx`, eg `call_xlIfzR1HqAW7xJPa3ExJSg3C`